.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug Report
+description: Something is not working
+title: "[Bug]: "
+labels: ["bug"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report! This issue type is to raise an issue with the way the Custom Elements Plugin (CEP) is working.
+- type: textarea
+  id: what-happened
+  attributes:
+    label: What happened?
+    description: Also tell us, what did you expect to happen?
+    placeholder: Tell us what you see!
```

Comment:
question:

description vs placeholder approach here vs other templates in this PR are different. Should the placeholder always be a different value than the "description"?